src/py/flwr/server/strategy/fedavg_android.py

```diff
@@ -40,7 +40,34 @@
 
 
 class FedAvgAndroid(Strategy):
-    """Configurable FedAvg strategy implementation."""
+    """Federated Averaging strategy.
+
+    Implementation based on https://arxiv.org/abs/1602.05629
+
+    Parameters
+    ----------
+    fraction_fit : Optional[float]
+        Fraction of clients used during training. Defaults to 0.1.
+    fraction_evaluate : Optional[float]
+        Fraction of clients used during validation. Defaults to 0.1.
```

Comment:
Those appear to be typos from the previous docstrings, it should be 1.0 in both cases